scripts/microbe_stage/microbe_editor/microbe_editor.as

```diff
@@ -960,6 +966,27 @@ class MicrobeEditor{
 
             templateOrganelles = newOrganelles;
 
+            // Grab render of player cell
+            auto node =  world.GetComponent_RenderNode(player);          
+
+
+            // Grab memnbrane to update scale value for absorber compound system
+            auto membraneComponent = world.GetComponent_MembraneComponent(player);
+
+            //! Change player species cell size depending on presence or not of nucleus
+            if(checkIsNucleusPresent()) {
```

Comment:
What is this for? Bacteria size is already correct, whats the point of this change???

Reply:
is not already merged in master branch.